### PR TITLE
Add integer validator

### PR DESCRIPTION
**Pull request by krolow**
As we have sanitizer to force be an integer, sometimes it will generate a NaN and often we would not like to deal with such NaN.

So this validator check if it is really an valid integer.

### CODE REVIEW

File: lib/validators.js [hunk 1]
```diff
@@ -144,6 +144,23 @@ validator.date = function(ctx, options, value) {
   }
 }
 
+validator.integer = function(ctx, options, value) {
+  if (typeof value === 'undefined' || value === null) return;
+
+  function isInteger(number) {
```
Comment (esnunes) [inline lib/validators.js]: I would like to suggest two modifications:
1. extract this function to outside the declaration of validator.integer;
1. at the moment the argument *number* is not used, modify the internal code to use number or change its name to *value* (which I prefer).

Comment (krolow) [inline lib/validators.js]: ohh yeah my bad, the value

Comment (krolow) [inline lib/validators.js]: alright commit ammend ;)

File: lib/validators.js [hunk 2]
```diff
@@ -144,6 +144,23 @@ validator.date = function(ctx, options, value) {
   }
 }
 
+validator.integer = function(ctx, options, value) {
+  if (typeof value === 'undefined' || value === null) return;
+
+  if (!isInteger(value)) {
```
Comment (krolow) [inline lib/validators.js]: @esnunes thinking in make if affirmative, what do you think...

```javascript
if (isInteger(value)) return;

var err = {validator: 'integer'};
addMessage(err, options);
return err;
```

Comment (esnunes) [inline lib/validators.js]: perfect

**Pull request merged**